src/_posts/2019-02-25-dynamodb-explained-part-1.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+asset-type: post
+name: 2019-02-25-dynamodb-explained-part-1
+title: DynamoDB explained - Part 1
+date: 2019-02-25 10:15:00 +00:00
+author: Bartomeu Galmés
+description: Deep diving on DynamoDB
+image: 
+    src: /assets/custom/img/blog/2019-02-25-dynamodb-explained-part-1/00-cloud.jpg
+tags: 
+- dynamodb
+- amazon web services
+- aws
+- databases
+- cloud
+
+---
+
+
+DynamoDB has been a bit of a buzzword recently, so after years working on relational and document databases I decided to give it a try. A lot of the assumptions I had made about DynamoDb before working with it turned out to be false, so I am hoping that this series of blog posts will help you decide if DynamoDB is appropriate for your next project.
+
+In this series of posts we will dive into DynamoDB and discover some of its quirks, which can differ a little from other databases. In this first post we will talk about what DynamoDB is and how it's approach to primary keys works. In the next post we will examine one of the most confusing aspects for people coming to DynamoDB from other databases - data retrieval - which works a little differently. We will see that, with a little help from indexes, we can achieve the most common use cases. The third and final part of this series will be focused on availability and scalability which are two big advantages of DynamoDB as AWS handles them for us.
```

Comment:
That's a matter of taste but I would break the lines explaining each post content, you can even use it for linking the future posts